docs/api/ref/DesktopAgent.md

```diff
@@ -376,21 +376,23 @@ await fdc3.open('myApp', context);
 ```ts
 raiseIntent(intent: string, context: Context, app?: TargetApp): Promise<IntentResolution>;
 ```
-Raises a specific intent against a target app.
+Raises a specific intent for resolution against apps registered with the desktop agent. 
 
-The desktop agent will resolve the correct app to target based on the provided intent name and context data.
+The desktop agent will resolve the correct app to target based on the provided intent name and context data. If multiple matching apps are found, the user may be presented with an app picker.
+Alternatively, the specific app to target can also be provided. A list of valid target applications can be retrieved via [`findIntent`](DesktopAgent#findintent).  
 
-If multiple matching apps are found, the user may be presented with an app picker.
-Alternatively, the specific app to target can also be provided (if known).
-
-Returns an `IntentResolution` object with a handle to the app that responded to the intent.
+Returns an `IntentResolution` object with details of the app that was selected to respond to the intent.
 
 If a target app for the intent cannot be found with the criteria provided, an `Error` with a string from the [`ResolveError`](Errors#resolverrror) enumeration is returned.
 
 #### Example
 
 ```js
-// find apps to resolve an intent to start a chat with a given contact
+// raise an intent for resolution by the desktop agent
+// a resolver UI will be displayed if more than one application can resolve the intent
```

Comment:
"a resolver UI _may_ be displayed..."

Reply:
Good catch, although I think its actually a _should_ as I see this in the compliance section of the docs:
>- Provide an API to all applications running in the context of the platform that meets the FDC3 API standards, including:
>    - Support for FDC3 Context Data and Intents standards
>    - Support for Intent and Context resolution using a resolver UI

https://fdc3.finos.org/docs/fdc3-compliance

Reply:
I agree, in which case we should update line 381 from _may_ to _should_ as well.